src/Form/Fields.php

```diff
@@ -23,7 +23,7 @@ class Fields extends Collection
      *
      * @param string $name
      * @param object|array $field
-     * @return self
+     * @return static
```

Comment:
The return value is inherited from `Toolkit\Collection::__set()`

```suggestion
     * @return $this
```